tests/filesystem/UploadTest.php

```diff
@@ -93,6 +93,30 @@ public function testAllowedFilesize() {
 		$result = $u1->load($tmpFile);
 		
 		$this->assertFalse($result, 'Load failed because size was too big');
+		
+		// check max file size set by app category
+		$tmpFileName = 'UploadTest-testUpload.jpg';
+		$tmpFilePath = TEMP_FOLDER . '/' . $tmpFileName;
+		file_put_contents($tmpFilePath, $tmpFileContent . $tmpFileContent);
+		
+		$tmpFile = array(
+			'name' => $tmpFileName,
+			'type' => 'image/jpeg',
+			'size' => filesize($tmpFilePath),
+			'tmp_name' => $tmpFilePath,
+			'extension' => 'jpg',
+			'error' => UPLOAD_ERR_OK,
+		);
+		
+		$v->setAllowedMaxFileSize(array('[image]' => '40k'));
+		$u1->setValidator($v);
+		$result = $u1->load($tmpFile);
+		$this->assertTrue($result, 'Load passed because file size was under limit');
```

Comment:
so `assertTrue` is saying "the first parameter MUST be `true` (ie: `$result === true`). it will fail if that is not the case and then the last param will be displayed in the test results.